charts/wildfly/README.md

```diff
@@ -146,8 +163,9 @@ The configuration to build the application image is configured in a `deploy` sec
 
 | Value | Description | Default | Additional Information |
 | ----- | ----------- | ------- | ---------------------- |
-| `deploy.annotations` | Map of `string` annotations that are applied to the deployment and its pod's `template` | - | [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) |
 | `deploy.enabled` | Determines if deployment-related resources should be created. | `true` | Set this to `false` if you do not want to deploy an application image built by this chart. |
+| `deploy.imagePullSecrets` | Names of secrets to pull the application image from an private image registry | - | [Kubernetes Documentation](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod) |
+| `deploy.annotations` | Map of `string` annotations that are applied to the deployment and its pod's `template` | - | [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) |
```

Comment:
Minor comment; I'm not sure why you changed the order here and I noticed the list is not sorted, following the Kubernetes API, I would expect a ordered list

Reply:
you're right. We should sort the list. If that's ok with you, I'll merge this PR and do another one to change the doc

Reply:
Sure, we should not delay integrating this because of that, my comments were just some tips to possible improvements

Reply:
Will be fixed in #85